### ISSUE TITLE
Handle Geolocation

### ISSUE THREAD

**Issue by TFilla** [SpaghettiCodeStudios/HowMuchWithTax#2]
Input of the location should allow basic user input in the form of a field, and should also take the location from the device if available.

Possibly helpful links:
https://dev.w3.org/geo/api/spec-source.html
http://www.w3schools.com/html/html5_geolocation.asp

These implementations look like they are based on latitude and longitude. So basically, here's what it looks like Geolocation needs:
- User input of City, Zip Code, or Coordinates
- Coordinate fetching from device
- Output of location onto a Google Map.
- Conversion of Latitude and Longtitude to Zip Code

**Comment by charlesabarnes** [comment]
So we want to do reverse Geocoding to get the lat/long into zip.   It appears as if the best way is to use Google's Geocoding API found [here](https://developers.google.com/maps/documentation/geocoding/intro#ReverseGeocoding)

**Comment by charlesabarnes** [comment]
Yesterday I attempted to set up a way to parse the JSON that the google geolocation API provides.   I was able to get the results, but apparently parsing the json with javascript is not supported.  For reference, we would want to implement something similar to [this](https://developers.google.com/maps/documentation/javascript/geocoding#ReverseGeocoding). 

Looks like obtaining lat/long was just the easy part.
